File: password generater in py,c,java/password.c
#include <stdio.h>
#include <stdlib.h>
void main()
{
    int digit,amount;
    printf("enter the size and amount");
    scanf("%d%d",&digit,&amount);
    
    char lower_case[]="qwertyuiopasdfghjklzxcvbnm";
    char upper_case[]="QWERTYUIOPASDFGHJKLZXCVBNM";
    char num[]="0123456789";
    char password[digit];
    for(int i=0;i<amount;i++){
        for(int j=0;j<digit;j++){
            int random=rand()%3;
            
            switch(random){
                
                case 0:
                password[j]=num[rand()%10];
                break;
                case 1:
                password[j]=lower_case[rand()%26];
                break;
                case 2:
                password[j]=upper_case[rand()%26];
                break;
                default:
                break;
            }
            
        }
        printf("%s\n",password);
    }
    
}

File: password generater in py,c,java/password.java
import java.util.Scanner;

class password
{
    public static void main(String args[])
    {
        Scanner input=new Scanner(System.in);
        System.out.println("Enter digit");
        int digit=input.nextInt();
        System.out.println("Enter amount");
        int amount=input.nextInt();

        String lower_case="qwertyuiopasdfghjklzxcvbnm";
        String upper_case="QWERTYUIOPASDFGHJKLZXCVBNM";
        String password="";

        for (int i=0;i<amount;i++) {
            password="";
            for(int j=0;j<digit;j++){
            int rand=(int)(3 * Math.random());
            
            switch (rand) {

                case 0:
                    password+=String.valueOf((int)(9 * Math.random()));
                    break;
                case 1:
                    rand=(int)(lower_case.length() * Math.random());
                    password+=String.valueOf(lower_case.charAt(rand));
                    break;

                case 2:
                    rand=(int)(upper_case.length() * Math.random());
                    password+=String.valueOf(upper_case.charAt(rand));
                    break;

                default :
                    break;
            }
            }
            System.out.println(password);}
        } 
    }